packages/playground/README.md

```diff
@@ -0,0 +1,3 @@
+# Cadl Playground
```

Comment:
I think this project might also need the thirdparty file to be generated.

Reply:
I think because this isn't distributed as a package, TPNs aren't required?

Reply:
ah maybe you are right here